app/views/govuk_publishing_components/components/_step_by_step_nav.html.erb

```diff
@@ -70,6 +71,7 @@
           <div class="gem-c-step-nav__panel js-panel" id="step-panel-<%= id %>-<%= step_index + 1 %>">
             <%
               in_substep = false
+              options[:step_nav_content_id] = step_nav_content_id
```

Comment:
Is `tracking_id` always populated? What happens if `step_nav_content_id` is false?